src/components/ScreenStackHeaderConfig.tsx

```diff
@@ -68,11 +89,19 @@ export const ScreenStackHeaderSearchBarView = (
 
 const styles = StyleSheet.create({
   headerSubview: {
-    position: 'absolute',
-    top: 0,
-    right: 0,
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'center',
   },
+  headerSubviewCenter: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexShrink: 1,
+  },
+  headerConfig: {
+    flexDirection: 'row',
+    width: '100%',
+    justifyContent: 'space-between',
```

Comment:
Don't we need to center subviews horizontally?

```suggestion
    justifyContent: 'space-between',
    alignItems: 'center',
```

Reply:
`justifyContent` centers the elements horizontally in this case.

Note that `justifyContent` centers the elements with respect to the "main axis". Due to `flexDirection: 'row'`, main axis is a horizontal axis.